src/wav2vec2/config.py

```diff
@@ -30,12 +31,22 @@ class Wav2Vec2Config:
     strides: list = field(default_factory=lambda: [5, 2, 2, 2, 2, 2, 2])
     conv_bias: bool = False
 
+    # spec augmentation arguments
+    apply_spec_augment: bool = False  # TODO: fix this
+    mask_time_prob: float = 0.05
+    mask_time_length: int = 10
+
     def __post_init__(self):
         if not (len(self.filter_sizes) == len(self.kernal_sizes) == len(self.strides)):
-            raise ValueError("len of filter_sizes, kernal_sizes, strides must much")
+            raise ValueError("len of filter_sizes, kernal_sizes, strides must match")
         if self.hidden_size % self.num_heads != 0:
             raise ValueError("Hidden size must be perfect multiple of num_heads")
 
+        assert self.loss_reduction in [
+            "mean",
+            "sum",
+        ], "Possible values of `loss_reduction` are `mean` & `sum`"
+
```

Comment:
If we are training on multi-core TPUs or even in a multi-GPU setting, we generally don't use these reduction methods from the losses. Refer [here](https://www.tensorflow.org/tutorials/distribute/custom_training).